js/battle.js

```diff
@@ -1030,11 +1038,19 @@ var Sprite = (function () {
 	Sprite.prototype.animDragIn = function (slot) {
 		if (this.battle.fastForward) return this.animSummon(slot, true);
 
+		if (!Tools.prefs('nopastgens') && this.battle.gen <= 4 && this.battle.gameType === 'doubles') {
+		this.x = (slot -.52 ) * (this.isBackSprite ? -1 : 1) * -55;
+		this.y = (this.isBackSprite ? -1 : 1) + 1;
+		this.statbarOffset = 0;
+		if (!this.isBackSprite) this.statbarOffset = 30 * slot;
+		if (this.isBackSprite) this.statbarOffset = -28 * slot;		
+		} else {
 		this.x = slot * (this.isBackSprite ? -1 : 1) * -50;
 		this.y = slot * (this.isBackSprite ? -1 : 1) * 10;
 		this.statbarOffset = 0;
 		if (!this.isBackSprite) this.statbarOffset = 17 * slot;
 		if (this.isBackSprite) this.statbarOffset = -7 * slot;
+			}
```

Comment:
The `}` needs to be unindented, and everything inside the `else { ... }` block needs to be indented.